system/ee/ExpressionEngine/View/addons/index.php

```diff
@@ -6,25 +6,38 @@
 			<?php if (!empty($updates) && count($updates) == 1) : ?>
 				<div id="alert_banner" class="alert">
 					<div class="alert__icon"><i class="fas fa-info-circle fa-fw"></i></div>
-					<div id="alert" href="" class="alert__close">
+					<div id="alert" class="alert__close">
 						<a id="alert_close" class="fas fa-times alert__close-icon"></a>
 					</div>
 					<p class="alert__title"> <br>
 					<p class="alert__title"><?php echo lang('update'); ?><br>
-						<?php echo lang('there_is'); ?> <strong><?= count($updates) ?></strong> <?php echo lang('single_update'); ?> <a id="ShowMe"><?php echo lang('show'); ?></a></p>
+						<?php
+						$there = lang('there_is');
+						$single = lang('single_update');
+						$count = count($updates);
+						echo sprintf($there, $count , $single);
+						?>
+						<a id="show_me"><?php echo lang('show'); ?></a>
+					</p>
+						
```

Comment:
Sorry for not being too clear. This is not what I meant. Been rether referring to having the string stored in lang file as one string, not two separate strings.

Here's example from another view file:

`<?php echo sprintf(lang('update_to_version'), '<br />' . $addon['update']); ?>`

and the lang row is `'Update to %s',`

so %s is getting replaced with whatever you pass as param